reana-ui/src/components/NotificationBell.js

```diff
@@ -0,0 +1,61 @@
+/*
+  -*- coding: utf-8 -*-
+
+  This file is part of REANA.
+  Copyright (C) 2022 CERN.
+
+  REANA is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+*/
+
+import { useSelector } from "react-redux";
+import { Popup, Icon, List } from "semantic-ui-react";
+
+import { getConfig, getUserQuota } from "~/selectors";
+import { getQuotaNotifications } from "~/pages/profile/components/Quota";
+import { healthMapping } from "~/util";
+
+export default function NotificationBell() {
+  const config = useSelector(getConfig);
+  const quota = useSelector(getUserQuota);
+
+  let notifications = getQuotaNotifications(quota, config.adminEmail);
+  const hasNotifications = !!notifications.length;
+  if (!hasNotifications) {
+    notifications.push({ body: "You have no notifications" });
+  }
+
+  return (
+    <Popup
+      trigger={
+        <Icon.Group size="large">
+          <Icon link name="bell outline" color="brown" />
+          {hasNotifications && (
+            <Icon corner="top right" name="circle" color="red" />
+          )}
+        </Icon.Group>
+      }
+      content={
+        <List divided relaxed>
+          {notifications.map((notification, index) => (
+            <List.Item key={index} as="a" href={notification.link}>
+              <List.Icon
+                verticalAlign="middle"
+                name={notification.icon}
+                color={healthMapping[notification.type]}
+              />
+              <List.Content>
+                <List.Header style={{ marginBottom: "0.2em" }}>
```

Comment:
I think we don't have a rule/convention for this. I personally think that if the component needs a single small css tweak it's ok to do it directly inside the jsx. If there are more than one styling change, then it should go to a dedicated file. Now it would look a bit weird to place this one statement inside a new file. WDYT?

Reply:
I think we can keep it like this, given that it is the only styling rule in the whole file. We will move it if we add more styling changes in the future